src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -165,4 +165,7 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
     this.message = `Are you sure you want to delete ${item.activity_name}?`;
     this.showModal = true;
   }
-}
\ No newline at end of file
+  dragEnded(event: any): void {
```

Comment:
Maybe, we could call this method as `resetDraggablePosition` or something similar.

Reply:
Is it a good idea or it can be resetDragPosition